ouroboros-network/src/Ouroboros/Network/NodeToClient.hs

```diff
@@ -290,44 +290,41 @@ ncSubscriptionWorker
     -> IO Void
 ncSubscriptionWorker
   sn
-  NetworkIPSubscriptionTracers
-    { nistSubscriptionTracer
-    , nistMuxTracer
-    , nistHandshakeTracer
-    , nistErrorPolicyTracer
+  NetworkSubscriptionTracers
+    { nsSubscriptionTracer
+    , nsMuxTracer
+    , nsHandshakeTracer
+    , nsErrorPolicyTracer
```

Comment:
In `cardano-sl`'s style the prefix would be `nst` not ns. But I'm not sure we have any rules for this in `ourboros-network`.

Reply:
I know, but we already have `nst` prefix in `NetworkServerTracers` :/